src/index.json

```diff
@@ -2218,7 +2218,7 @@
                     "summary": "Provides a preview for upcoming storage features.",
                     "version": "0.2.9"
                 },
-                "sha256Digest": "211afaf23a6340671fee8a4b9e7c982f28b9ec9d1bf2e403fcc9de08c561ab7b"
+                "sha256Digest": "880e01de0fab8893770497ef9410559ae223a1f09dbd6a23712226ab4e2d5ecb"
```

Comment:
You also change index.json for storage-preview?

Reply:
resolved offline.